client/driver/docker.go

```diff
@@ -970,8 +970,11 @@ func (d *DockerDriver) createContainerConfig(ctx *ExecContext, task *structs.Tas
 		VolumeDriver: driverConfig.VolumeDriver,
 	}
 
-	// Windows does not support MemorySwap #2193
-	if runtime.GOOS != "windows" {
+	// Windows does not support MemorySwap/MemorySwappiness #2193
+	if runtime.GOOS == "windows" {
+		hostConfig.MemorySwap = 0
+		hostConfig.MemorySwappiness = -1
```

Comment:
I think you have the 0 and -1 switched judging by the API docs here: https://docs.docker.com/engine/api/v1.31/#operation/ContainerUpdate (search for "swappiness")

Reply:
One would wish this was documented better :) I actually brought up a windows server and this is the only config that works. -1 disables snappiness since the valid inputs are 0-100 and MemorySwap has to be 0 as well since there is no support for any swap.